### PR TITLE
MultiFileCachingCompiler does not set up its internal LRU cache correctly

### PR DESCRIPTION
**Pull request by leonardoventurini**
Pin `lru-cache` version to `6.0.0` everywhere it is used and fix its usages, could not update it to `10.2.2` as it broke the bundling process

Addresses https://github.com/meteor/meteor/issues/13119

[Metaboard](https://metaboard.cc/b/665f00ef4cd72c3e113ad8ff)

### CODE REVIEW

**Review by nachocodoner — DISMISSED** [review]

**Comment by StorytellerCZ** [comment]
Would be nice to upgrade to the latest version soon.

**Comment by leonardoventurini** [comment]
> Would be nice to upgrade to the latest version soon.

Will try my hand at updating it to v10 again before merging, let's see if I can make the memory issue go away

**Comment by leonardoventurini** [comment]
The automated tests broke because I changed a dependency in the bundle itself, and tests don't regenerate the bundle it seems.

**Comment by StorytellerCZ** [comment]
Yes, the bundle will have to be updated.

File: npm-packages/meteor-babel/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@meteorjs/babel",
   "author": "Meteor <dev@meteor.com>",
-  "version": "7.20.0-beta.4",
+  "version": "7.20.0-beta.5",
```
Comment (StorytellerCZ) [inline npm-packages/meteor-babel/package.json:4]: Not related to this PR, but just reminding everyone that we must not forget to change this version from beta to actual version once v3 is released.

**Review by StorytellerCZ — APPROVED** [review]

**Pull request merged**